integreat_cms/cms/views/events/event_form_view.py

```diff
@@ -234,6 +235,26 @@ def post(self, request, **kwargs):
                 event_translation_form.instance.event.translations.filter(
                     language__in=languages
                 ).update(status=status.DRAFT)
+
+            # Show a message that the slug was changed if it was not unique
+            if user_slug and user_slug != event_translation_form.cleaned_data["slug"]:
+                other_translation = EventTranslation.objects.filter(
+                    event__region=region, slug=user_slug, language=language
+                ).first()
+                other_translation_link = other_translation.backend_edit_link
+                messages.warning(
+                    request,
+                    _(
+                        "The slug was changed from '%(user_slug)s' to '%(slug)s', because '%(user_slug)s' is already used by <a href='%(link)s' class='underline hover:no-underline'>%(translation)s</a> or one of its previous versions"
+                    )
+                    % {
+                        "user_slug": user_slug,
+                        "slug": event_translation_form.cleaned_data["slug"],
+                        "link": other_translation_link,
+                        "translation": other_translation,
+                    },
```

Comment:
```suggestion
                        "The slug was changed from '{user_slug}' to '{slug}', because '{user_slug}' is already used by <a href='{link}' class='underline hover:no-underline'>{translation}</a> or one of its previous versions"
                    ).format(
                        user_slug=user_slug,
                        slug=event_translation_form.cleaned_data["slug"],
                        link=other_translation_link,
                        translation=other_translation,
                    ),
```
I find the `%(var)s` syntax a bit harder to read than `{var}` in the translation file, so I would suggest to use the newer format string syntax everywhere except logging messages (where the lazy loading of the `%` format string actually means an advantage)...